.zenodo.json

```
@@ -162,9 +172,7 @@
             "name": "Rajul Srivastava"
         },
         {
-            "name": "Shane Maloney",
-            "orcid": "0000-0002-4715-1805",
-            "affiliation": "Trinity College Dublin / Dublin Institute for Advanced Studies"
+            "name": "Samuel Bennett"
```

@CyclingNinja you got an orcid?
```suggestion
            "name": "Samuel Bennett",
            "affiliation": "Aperio Software Ltd."
```